### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by StefanoFiumara** [StefanoFiumara/harry-potter-tcg#100] [General]
# ⚡ Harry Potter TCG ⚡

## Version 0.2.0
This is the second preview build of the game! All features from the v0.2.0 section of the roadmap are here:

* Complex Spells (multiple input steps, targeting cards in deck/discard piles)
* Basic Items
* Basic Characters
* Simple Deck Editor
* Community Suggestions from v0.1.0
* UI Enhancements
* HD Resolution Cards
* Cleaned up animations

This game client is dedicated to the HP TCG Revival community, check them out on Facebook and Discord:
* https://www.facebook.com/groups/harrypottertcg
* http://discord.harrypottertcg.com

#### High Quality Card Graphics
* Enjoy glorious high quality card graphics courtesy of the HP TCG Revival Team

![image](https://user-images.githubusercontent.com/2746784/120750146-7da06180-c4cb-11eb-915e-12f87dcb8b90.png)

#### Deck Builder Screen
* Add cards to your deck by clicking on them in the main panel
* Remove cards from your deck by clicking on them in the deck side panel on the right
* Set your starting character by right-clicking a character card in the main panel
* Use the search bar and lesson/card type filters on the top of the screen to quickly find your favorite cards
* You will not be able to enter the gameplay screen without first building a valid deck

![image](https://user-images.githubusercontent.com/2746784/120749572-73ca2e80-c4ca-11eb-9631-42274cf800e6.png)

#### New Gameplay Features
* The gameplay UI has been slightly modified to more clearly display remaining card counts in each player's deck
* Players can now view each other's discard piles by clicking the "View" near each pile
* Players can now forfeit a match by clicking the "Forfeit" button (don't worry, it asks if you're sure 😉 )
* Cards with in-play abilities (characters, items) can be activated by clicking on the card during your turn

![image](https://user-images.githubusercontent.com/2746784/120749935-0bc81800-c4cb-11eb-9c40-2eab342937ee.png)

## Installation
There is no installation required. To play the game, simply unzip the contents of `hp-tcg-v0.2.0.zip` to a new folder on your PC, then double click `Harry Potter TCG.exe` to start the game.

## Feedback
To leave general feedback, suggestions or to report a bug, you can use the following form: https://forms.gle/vcL9rG7ewKpWfYHNA
You may also open an issue on Github if you have problems.

Thank you for checking out the game!

<hr /><em>This discussion was created from the release <a href='https://github.com/StefanoFiumara/harry-potter-tcg/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by DG-Kneel** [comment]
Where can I download the EXE? I can't find it on here.

**Comment by vdsams** [comment]
Hey,

What are you trying to access? Everything on the repo should be public.

If you want to DM on discord (vdsams) or FB that might the easier to talk.

Stay safe,
Sammy

**Reply by DG-Kneel** [reply]
I am trying to find the EXE so I can play. I saw Stefano's reply below...but sadly I cannot find the file he is referencing.

**Reply by DG-Kneel** [reply]
FOUND IT

**Comment by StefanoFiumara** [comment]
@DG-Kneel 

Sorry for the late reply, in the releases page, click on `hp-tcg-v0.2.0.zip` at the very bottom of the post, you can find the executable after extracting that folder anywhere on your PC.

![image](https://github.com/user-attachments/assets/ebbc5bd5-011f-4bf6-8230-c883efd47b98)

**Reply by DG-Kneel** [reply]
I am looking for this...but I don't see it. Maybe I am just blind or stupid. Sorry. I even did a search on GITHUB for hp-tcg-v0.2.0.zip and it says ZERO hits.

**Reply by DG-Kneel** [reply]
![image](https://github.com/user-attachments/assets/20ca0925-a44e-4806-86e7-1580767391e3)

**Reply by DG-Kneel** [reply]
That's all I see...am I in the wrong place?

**Reply by StefanoFiumara** [reply]
On the right side of that screen, there should be a section called "releases", you should find the download links under the v0.2.0 version tag on that page

![image](https://github.com/user-attachments/assets/98dabd76-47bf-44f6-9c18-d39538fdb227)

**Reply by DG-Kneel** [reply]
FOUND IT! Thank you!!!!!

**Comment by DG-Kneel** [comment]
Stefano,
Sorry for the additional questions, but is there a way to add more cards to the game? Have additional cards been made?
    On Tuesday, January 14, 2025 at 06:44:25 AM GMT+9, Stefano Fiumara ***@***.***> wrote:  
 
 


@DG-Kneel

Sorry for the late reply, in the releases page, click on hp-tcg-v0.2.0.zip at the very bottom of the post, you can find the executable after extracting that folder anywhere on your PC.

image.png (view on web)

**Comment by StefanoFiumara** [comment]
Hey @DG-Kneel sorry for the late reply.

There is a way to add additional cards, but you will have to download the source code and open the project using the Unity game engine, there are some editor plugins included alongside the project that would make this relatively simple. It may be a bit daunting however if you haven't used Unity before or if you aren't familiar with programming.

I haven't worked on this project in a long time, mainly due to some remaining game mechanics that I find very difficult to implement, as this card game was designed to be played over a tabletop and requires a lot of interactions which are challenging to translate into engine driven rules.

Furthermore, the HPTCG community has settled onto other online platforms to play the game on, I'd like to eventually call this project done, but I don't know that I will ever get to adding every single card that currently exists.
